Makefile

```diff
@@ -24,6 +24,10 @@ verify-podnetworkconnectivitychecks:
 	$(MAKE) -C pkg/operator/connectivitycheckcontroller verify
 
 test-e2e-encryption: GO_TEST_PACKAGES :=./test/e2e-encryption/...
+test-e2e-encryption: GO_TEST_FLAGS += -v
+test-e2e-encryption: GO_TEST_FLAGS += -timeout 4h
```

Comment:
4h for a single unit test is quite a long timeout :)

Reply:
This is mostly copied from https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/Makefile#L45-L48.

And there nothing unit about this test besides the fact that we invoke "test-unit" from build-machinery. The test creates 7 different keys: https://github.com/openshift/library-go/blob/master/test/e2e-encryption/encryption_test.go#L366-L371 which should result in at least 21 revisions if we include the migrations, which can take a while :sweat_smile: